README.md

```diff
@@ -68,6 +68,10 @@ see [`GsonBuilder.disableJdkUnsafe()`](https://javadoc.io/doc/com.google.code.gs
 
 Older Gson versions may also support lower API levels, however this has not been verified.
 
+### R8 / ProGuard
+
+See the details in the related section in [Troubleshooting guide](Troubleshooting.md#r8--proguard).
```

Comment:
```suggestion
See the details in the related section in the [Troubleshooting guide](Troubleshooting.md#r8--proguard).
```

Maybe it would also be better to remove this below the "Documentation" section, since it is more specific?

Reply:
I document the R8 / ProGuard section explicitly as I want to warn users like https://github.com/square/retrofit?tab=readme-ov-file#r8--proguard and OkHttp did.

Reply:
Yes that is fine and a good thing. I was just wondering whether it would be better to have the (existing) "Documentation" section first in the README:
> ...
>
> ## Documentation
> - API Javadoc: Documentation for the current release
> - ...
>
> ## ProGuard / R8
> See the details in the related ...

Because this is a more specific aspect of the documentation.

Reply:
Reasonable! 495aa1b19e0f6526eaf323d23232361989961293